osm-seed/values.yaml

```diff
@@ -61,7 +61,7 @@ db:
   enabled: true
   image:
     name: 'developmentseed/osmseed-db'
-    tag: '0.1.0-n377.hd099819'
+    tag: '0.1.0-n423.hdb62b0d'
```

Comment:
we really should avoid committing these changes to the repo. these should all ideally just be something like `chartpress_replace` in the repository?

Reply:
Looks like it was set  when i ran `chartpress` 😬 , Do  you want me to remove that?